src/docs/sdk/event-payloads/index.mdx

```diff
@@ -236,6 +236,43 @@ import "./properties/tags.mdx"
 For information about overriding grouping see [Customize Grouping with
 Fingerprints](https://docs.sentry.io/data-management/event-grouping/).
 
+`errors`
+
+: A list of errors in capturing or handling this event. This provides meta data
+  about event capturing and processing itself, not about the error or
+  transaction that the event represents.
+
+  This list is primarily populated by Sentry when receiving and processing the
+  event. SDKs are only encouraged to add entries here if there are severe
+  conditions that Sentry cannot detect by inspecting the remaining payload.
+
+  Errors must contain a required `type` field, which can be one of the types
+  declared in the Sentry [EventError] model. If there is no applicable type
+  variant, consider [opening an issue] to suggest the addition.
+
+  Apart from types, any attribute is valid. There is a set of conventions for
+  common error attributes:
+
+- `name`: A string declaring the path to the payload field that causes or
+  exhibits the error. For example `modules[0].name`.
+- `value`: The original value of a field that causes or exhibits the error.
+
+
+```json
+{
+  "errors": [
+    {
+      "type": "unknown_error",
+      "path": "/var/logs/errors.log.1",
+      "details": "Failed to read attachment"
+    }
+  ]
+}
+```
```

Comment:
`l`: Above we state that `name` and `value` are conventions, but we miss them in here.

Reply:
Good point. This was intentional to show that there is no obligation to send them (and it often doesn't make sense), but rather if they are specified, they should have a certain meaning.

Reply:
@philipphofmann updated the sentence introducing conventions. Does it read better now?

Reply:
Yes, is better now. Thanks 👍🏼